add-on/src/popup/quick-upload.js

```diff
@@ -61,17 +71,20 @@ async function processFiles (state, emitter, files) {
     }
     const { ipfsCompanion } = await browser.runtime.getBackgroundPage()
     const uploadTab = await browser.tabs.getCurrent()
-    const { streams, totalSize } = files2streams(files)
-    progressHandler(0, totalSize, state, emitter)
+    const streams = files2streams(files)
     emitter.emit('render')
-    const wrapFlag = (state.wrapWithDirectory || streams.length > 1)
+    const wrapFlag = streams.length > 1
     const options = {
-      progress: (len) => progressHandler(len, totalSize, state, emitter),
-      wrapWithDirectory: wrapFlag,
-      pin: state.pinUpload
+      wrapWithDirectory: wrapFlag
```

Comment:
The `ipfs.add` command  adds a low level pin by default.
We use MFS instead of low level pins, so we need to disable  it here:


```suggestion
      wrapWithDirectory: wrapFlag,
      pin: false // we use MFS for implicit pinning instead
```